ROMUtils.cpp

```diff
@@ -41,6 +41,10 @@ namespace ROMUtils
     unsigned char *tmpCurrentFile;
     unsigned int tmpCurrentFileSize;
     QString tmpROMFilePath;
+    struct ROMFileMetaData CurrentROMMetaData;
+    struct ROMFileMetaData TemROMMetaData;
```

Comment:
TempROMMetadata (missing the p) also let's name these as "Metadata" instead of MetaData (metadata is one word)